spring/boot2-webflux-autoconfigure/build.gradle

```diff
@@ -37,6 +37,7 @@ task generateSources(type: Copy) {
     include '**/ArmeriaServerConfigurator.java'
     include '**/ArmeriaSettings.java'
     include '**/CustomAlias*KeyManager*.java'
+    include '**/DocServiceConfigurator.java'
```

Comment:
Could you include `ArmeriaBeanPostProcessorConfiguration` by changing
`include '**/*ArmeriaBeanPostProcessor.java'` to `include '**/*ArmeriaBeanPostProcessor*.java'`
please?

Reply:
Also could group the classes with the package so that we can easily recognize what we missed please?
like:
```
// Copy from /internal/spring package.
include '**/ArmeriaConfigurationNetUtil.java'
include '**/ArmeriaConfigurationUtil.java'
include '**/CustomAlias*KeyManager*.java'

// Copy from /spring package.
```